### PR TITLE
add confirm to hard reset on the crash screen

### PR DESCRIPTION
**Pull request by NastiaS**
I tested brining up a confirmation modal for the hard reset by throwing an error from the init and 2 components inside the app. see if there are other scenarios to test

### CODE REVIEW

File: src/components/RenderError.js [hunk 1]
```diff
@@ -22,11 +25,33 @@ type Props = {
...
+> {
   state = {
     isHardResetting: false,
+    isHardResetModalOpened: false,
   }
 
+  handleOpenHardResetModal = () => this.setState({ isHardResetModalOpened: true })
+  handleCloseHardResetModal = () => this.setState({ isHardResetModalOpened: false })
+
+  handleHardReset = async () => {
```
Comment (meriadec) [inline src/components/RenderError.js]: you forgot to remove the other `handleHardReset` method :p

Comment (NastiaS) [reply src/components/RenderError.js]: I certainly did 🙄and neither lint nor something else complained..

File: src/components/RenderError.js [hunk 2]
```diff
@@ -14,6 +14,9 @@ import ExportLogsBtn from 'components/ExportLogsBtn'
 import Box from 'components/base/Box'
 import Space from 'components/base/Space'
 import Button from 'components/base/Button'
+import IconTriangleWarning from 'icons/TriangleWarning'
+import ConfirmModal from './base/Modal/ConfirmModal'
```
Comment (meriadec) [inline src/components/RenderError.js]: better to keep consistency with `components/base/Modal/ConfirmModal` ..

Comment (meriadec) [reply src/components/RenderError.js]: lets merge and we'll change them all later

Comment (NastiaS) [reply src/components/RenderError.js]: not again 😁

**Pull request merged**